src/pyskindose/geom_calc.py

```diff
@@ -48,22 +47,19 @@ def calculate_field_size(field_size_mode, data_parsed, data_norm):
     return FS_lat, FS_long
 
 
-def position_geometry(
+def position_patient_phantom_on_table(
         patient: Phantom,
         table: Phantom,
         pad: Phantom,
         pad_thickness: Any,
         patient_offset: List[int],
         patient_orientation: c.PATIENT_ORIENTATION_HEAD_FIRST_SUPINE
                     ) -> None:
-    """Manual positioning of the phantoms before procedure starts.
+    """Manual positioning of the phantom upon the patient support table.
```

Comment:
Is it correct to call it a _manual positioning_?

Reply:
Nope. I have updated this sentence in b9fe4e9d922ae722ddaf41a93f0b0450a67aa60a